charts/spire/README.md

```diff
@@ -7,6 +7,48 @@ A Helm chart for deploying the complete Spire stack including: spire-server, spi
 
 **Homepage:** <https://github.com/spiffe/helm-charts/tree/main/charts/spire>
 
+## Install notes
+
+To do a quick non production install suitable for quick testing in something like minikube:
+
+```shell
+kubectl create namespace spire-system
+helm install -n spire-system spire-crds --repo https://spiffe.github.io/helm-charts-hardened/
+helm install -n spire-system spire --repo https://spiffe.github.io/helm-charts-hardened/
+```
+To customize, start with a base values file and edit as needed:
+```shell
+curl -o your-values.yaml https://raw.githubusercontent.com/spiffe/helm-charts-hardened/main/examples/production/example-your-values.yaml
+```
+Then:
+```shell
+helm install -n spire-system spire --repo https://spiffe.github.io/helm-charts-hardened/ -f your-values.yaml
```

Comment:
Helm hasn't done a great job explaining you shouldn't edit charts/spire/values.yaml directly, but instead provide your own values file. This provides an example of how to do it.